tests/text/test_bleu.py

```diff
@@ -91,7 +94,17 @@ def test_bleu_score_class(self, ddp, dist_sync_on_step, preds, targets, weights,
 
     def test_bleu_score_functional(self, preds, targets, weights, n_gram, smooth_func, smooth):
         metric_args = {"n_gram": n_gram, "smooth": smooth}
-        nltk_metric = partial(corpus_bleu, weights=weights, smoothing_function=smooth_func)
+
+        def nltk_metric(list_of_references, hypotheses, weights=weights, smoothing_function=smooth_func, **kwargs):
```

Comment:
@stancld  and @Borda 

so nltk's `corpus_bleu` takes tokenized texts as input rather than strings: https://www.nltk.org/_modules/nltk/translate/bleu_score.html

I tried to put a wrapper around `corpus_bleu` to ensure we can validate the result between PTL's bleu vs. NLTK's bleu, however I have been getting ` AttributeError: Can't pickle local object 'TestBLEUScore.test_bleu_score_class.<locals>.nltk_metric'` as error when it was running `run_class_metric_test`. I presume it's because it didn't like the new `nltk_metric` wrapper. Do you all have other suggestion to implement this?

Reply:
disregard. solved this.